src/server/auth/cmds/configure.go

```diff
@@ -37,7 +37,7 @@ func GetConfig() *cobra.Command {
 				fmt.Println("no auth config set")
 				return nil
 			}
-			output, err := json.Marshal(resp.Configuration)
+			output, err := json.MarshalIndent(resp.Configuration, "", "  ")
```

Comment:
Is there a reason we don't want to indentation? Seems like it makes it more readable which is what this command is meant for.

Reply:
Yeah, question is confusing because I was very confused when I asked it, somehow mixed up which was the new code and which was the old code. I get it now.